modepy/matrices.py

```diff
@@ -160,10 +161,10 @@ def differentiation_matrices(basis, grad_basis, nodes, from_nodes=None):
 
     :arg basis: A sequence of basis functions accepting arrays
         of shape *(dims, npts)*,
-        like those returned by :func:`modepy.simplex_onb`.
+        like those returned by :func:`modepy.orthonormal_basis_for_shape`.
```

Comment:
```suggestion
        like those returned by :func:`modepy.Basis.functions`.
```
? Since `orthonormal_basis_for_shape` seems to return a `Basis` class.